tests/test_ethanchlebowski.py

```diff
@@ -0,0 +1,65 @@
+from recipe_scrapers.ethanchlebowski import EthanChlebowski
+from tests import ScraperTest
+
+
+class TestEthanChlebowskiScraper(ScraperTest):
+
+    scraper_class = EthanChlebowski
+
+    def test_host(self):
+        self.assertEqual("ethanchlebowski.com", self.harvester_class.host())
+
+    def test_author(self):
+        self.assertEqual("Ethan Chlebowski", self.harvester_class.author())
+
+    def test_title(self):
+        self.assertEqual("Huevos Rancheros", self.harvester_class.title())
+
+    def test_category(self):
+        self.assertEqual(None, self.harvester_class.category())
+
+    def test_total_time(self):
+        self.assertEqual(13, self.harvester_class.total_time())
+
+    def test_yields(self):
+        self.assertEqual("0 servings", self.harvester_class.yields())
```

Comment:
Huh, interesting.  In the recipe HTML, the yield is described using a schema.org field called `yield`, whereas we expect to read it from `recipeYield`.

Both seem to be valid field names according to schema.org: https://schema.org/Recipe

Could you try changing this line: https://github.com/hhursev/recipe-scrapers/blob/00a460d2c20efe6d17485ca2c3479ea0131693af/recipe_scrapers/_schemaorg.py#L128

To this:

```python
yield_data = self.data.get("recipeYield") or self.data.get("yield")
```

...and then let's see if that fixes this result.  There's a chance it will affect (and hopefully improve) other scrapers, too.

Reply:
Fixed

Reply:
:100: nice, thank you